qemu/qmp.go

```diff
@@ -178,6 +178,28 @@ type MemoryDevices struct {
 	Type string            `json:"type"`
 }
 
+// CPUInfo represents information about each virtual CPU
+type CPUInfo struct {
+	CPU      int           `json:"CPU"`
+	Current  bool          `json:"current"`
+	Halted   bool          `json:"halted"`
+	QomPath  string        `json:"qom_path"`
```

Comment:
JSON format not appropriate IMO, we should use dashes `-` instead of underscores `_` in order to be consistent with the existing code already:
`json:"qom-path"`

Reply:
It's a defined QMP response, we can only choose to use but can't modify it as we want.